# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dog/manpage-touchup.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

The correct syntax of the -w option does not have a space between 
the -w and the cols ... the manpage seems to imply differently though 
so let's fix it.

http://bugs.gentoo.org/show_bug.cgi?id=69660
This patch comes from Gentoo, author is the gentoo maintainer of dog.
--- ./dog.1
+++ ./dog.1
@@ -8,3 +8,3 @@
 .B dog
-[\-AbBeEnstTuv] [\-w cols] [\-l lines] 
+[\-AbBeEnstTuv] [\-w<cols>] [\-l lines] 
 [\-\-show\-all] [\-\-number\-nonblank] [\-\-no\-blanks] [\-\-bind=port]
@@ -64,3 +64,3 @@
 .TP
-.I \-w n
+.I \-w<n>
 Print first n characters of each line (default=80)
